baseReportLib/report_params.rc

```diff
@@ -95,3 +95,9 @@
 #define RISM_Include_Data_Threshold_Hepatocytes_DESC_TEXT "If the column is included and the number of hepatocytes related to this infection are greater than or equal to this value, then the row of data will be included."
 #define RISM_Include_Data_Threshold_IRBC_DESC_TEXT "If the column is included and the number of IRBCs related to this infection are greater than or equal to this value, then the row of data will be included."
 #define RISM_Include_Data_Threshold_Gametocytes_DESC_TEXT "If the column is included and the number of gametocytes related to this infection are greater than or equal to this value, then the row of data will be included."
+
+#define RVM_Must_Be_In_State_DESC_TEXT "A comma-separated list of vector states for which you want to record the migration. Note: Only STATE_MALE, STATE_ADULT, STATE_INFECTED, STATE_INFECTIOUS migrate."
```

Comment:
Hmm.  I'm not sure we say "comma-separated" anywhere else.  I think we just say "list"

Reply:
Probably don't need "Note:"